### ISSUE TITLE
Makefile relies on pip's `--use-wheel` option which was removed in pip 10.0

### ISSUE THREAD

**Issue by hbcondo** [hypothesis/h#4970]
### Steps to reproduce
1. Complete all installation steps as specified on [Website dev install instructions](https://h.readthedocs.io/en/latest/developing/install/) 
2. Run `make dev` [command](https://h.readthedocs.io/en/latest/developing/install/#running-h)

### Expected behaviour
Development server starts on port 5000 (http://localhost:5000)

### Actual behaviour
Error returned:
```
no such option: --use-wheel
Makefile:74: recipe for target '.pydeps' failed
make: *** [.pydeps] Error 2
```

### Browser/system information
Distributor ID: Ubuntu
Description:    Ubuntu 17.10
Release:        17.10
Codename:       artful

**Comment by robertknight** [comment]
Confirmed. pip 10.0 was recently release which removed the `--use-wheel` option: https://pip.pypa.io/en/stable/news/#b1-2018-03-31

**Comment by robertknight** [comment]
This was fixed by bcac50a1eba28cc607a4ee904ee7f9e4646ddfa1
